multinet/swagger/set_metadata.yaml

```diff
@@ -0,0 +1,35 @@
+Set the metadata of a table.
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table"
+  - name: metadata
+    in: body
+    description: The metadata to set (overwrites existing data)
```

Comment:
Seems to update, not just overwrite everything

Reply:
Related to the other comment, but it overwrites the table metadata.